ui/selectors/selectors.js

```diff
@@ -1326,6 +1390,29 @@ export const getInsightSnaps = createDeepEqualSelector(
   },
 );
 
+/**
+ * Get a selector that returns insight snaps metadata (name and description).
+ *
+ * @param {object} state - The Redux state object.
+ * @returns {object} An object with snapId keys and values of snap names and descriptions.
+ */
+export const getInsightSnapsMetadata = createDeepEqualSelector(
```

Comment:
Why do we need this and `getSnapsMetadata`? 🤔

Reply:
You're right lol, not needed. Removed!